security-response-policies.md

```diff
@@ -0,0 +1,29 @@
+# Security Response Policy Guidelines
+
+The [Project Proposal Template](project-submission-template.md) specifies that
+a project should state what its security response policy is, if any, when
+being submitted to the eBPF Foundation.
+
+Each Technical Project should have a written security response policy, but the
+details may vary by project.  A written policy is not a prerequisite for
+submitting a project to the eBPF Foundation, but once accepted, projects are expected
+to have some documented disclosure process.  A security response policy
+should include an inbound disclosure process, and an outbound disclosure
+process.  
+
+## Inbound Disclosure Process
+
+Questions to consider include:
+
+1. How should a vulnerability be disclosed to the project?
+2. Is anonymous disclosure permitted, and if so, how?
+
+## Outbound Disclosure Process
+
+Questions to consider include:
```

Comment:
Most of these questions are predicated on (1) being a nonzero set. I think that's fine, though it does also to a degree prescribe that the expectation is that there is a more sophisticated process. Is it OK to respond "nobody" for (1)?

Maybe one other "outbound" question might be:
- Does the project publish public advance warning regarding upcoming security-relevant releases (dates only, no details)?